examples/user-management/react-user-management/src/Account.jsx

```diff
@@ -43,7 +43,7 @@ export default function Account({ session }) {
       id: user.id,
       username,
       website,
-      avatarUrl,
+      avatar_url,
```

Comment:
```suggestion
      avatar_url: avatarUrl,
```

Reply:
Thanks! Appreciate it :)

Haven't tested, but looks like you should need this to get it working?